kibana-reports/public/components/context_menu/context_menu_ui.js

```diff
@@ -142,7 +142,7 @@ export const popoverMenuDiscover = (savedObjectAvailable) => {
     </div>
     <div data-focus-lock-disabled="disabled">
        <div class="euiPanel euiPopover__panel euiPopover__panel--bottom euiPopover__panel-isOpen euiPopover__panel-withTitle" aria-live="assertive" role="dialog" aria-modal="true" aria-describedby="i199c7fc0-f92e-11ea-a40d-395bfe9dc89a" style="top: 97px; left: 255.583px; z-index: 2000;" id="reportPopover">
-          <div class="euiPopover__panelArrow euiPopover__panelArrow--bottom" style="left: 15.9417px; top: 0px;">
+          <div class="euiPopover__panelArrow euiPopover__panelArrow--bottom" style="left: 30.9417px; top: 0px;">
```

Comment:
Is this necessary?  I didn't see this change in your PR to `dev`

Reply:
Yeah- in 7.9.1 the `Discover` reporting menu was not lined up over the `Reporting` nav menu item, it's almost not under it at all. 
![Screen Shot 2021-01-11 at 9 27 54 AM](https://user-images.githubusercontent.com/53581635/104216779-5e574980-53ef-11eb-922e-e7d0530d175f.png)